sale_product_set/models/product_set.py

```diff
@@ -1,11 +1,18 @@
 # -*- encoding: utf-8 -*-
-from openerp import fields, models, _
+from openerp import api, fields, models, _
 
 
 class ProductSet(models.Model):
     _name = 'product.set'
     _description = 'Product set'
 
-    name = fields.Char(_('Name'), help=_('Product set name'), required=True)
+    name = fields.Char(u"Name", help=u"Product set name", required=True)
     set_line_ids = fields.One2many(
-        'product.set.line', 'product_set_id', _(u"Products"))
+        'product.set.line', 'product_set_id', u"Products", copy=True)
+
+    @api.one
```

Comment:
I'm afraid it's not possible in current ORM implementation: https://github.com/odoo/odoo/blob/8.0/openerp/models.py#L4853, because the argument is only an ID and the return is also only one ID. In v9 the situation is the same (https://github.com/odoo/odoo/blob/9.0/openerp/models.py#L4914), so maybe in a future?